apps/roam/src/components/DiscourseContext.tsx

```diff
@@ -345,17 +349,28 @@ export const ContextContent = ({ uid, results }: Props) => {
     }));
   }, []);
 
-  const onRefresh = useCallback(() => {
-    setRawQueryResults({});
-    getDiscourseContextResults({
-      uid,
-      onResult: addLabels,
-      ignoreCache: true,
-    }).finally(() => setLoading(false));
-  }, [uid, setRawQueryResults, setLoading, addLabels]);
+  // never set ignoreCache to true directly,
+  // as it may be negated by the useCallback cache.
```

Comment:
`useCallback` does not “cache” results or arguments. It only memoizes the function reference (identity).